jazzhands/__main__.py

```diff
@@ -141,23 +207,32 @@ def main(argv):
         )
         if os.path.isdir(dirname) and has_settings:
             project_dir = dirname
-    assert project_dir
+    assert project_dir  # If we can't find the location of the project's "main" package, abort
 
     # Look through all the Python paths for packages with static files in them
     for path in sys.path:
         for root, dirs, files in os.walk(path):
+            # Skip some obvious irrelevant directories
             if ".tox" in root or 'node_modules' in root or ".git" in root:
```

Comment:
This might be a little over-broad, skipping any directory that has .tox etc as a substring.  Maybe something like:

    root_parts = root.split('/')
    if '.tox' in root_parts or 'node_modules' in root_parts or '.git' in root_parts:
         ...

?

Reply:
I forgot that you can modify `dirs` in place to exclude paths from being walked down at all!

`dirs[:] = [d for d in dirs if d not in exclude_dirs]`

and as a bonus this step now runs way faster by skipping `node_modules` and `.git` entirely